closed/src/java.base/share/native/libjncrypto/NativeCrypto.c

```diff
@@ -276,7 +305,7 @@ JNIEXPORT jint JNICALL Java_jdk_crypto_jniprovider_NativeCrypto_loadCrypto
     /* Determine the version of OpenSSL. */
     OSSL_version_t* OSSL_version;
     const char * openssl_version;
-    int ossl_ver;
+    long ossl_ver = 0;
```

Comment:
The type should be `jlong` to match `extractVersionToJlong()`.

Reply:
Edited.